bin/hook-reference/data.js

```diff
@@ -4,9 +4,11 @@ const { parse } = require( 'comment-parser/lib' );
 const { relative, resolve } = require( 'path' );
 const chalk = require( 'chalk' );
 
+const dataTypes = [ 'filter', 'slotFill' ];
```

Comment:
I think we'll need to add `action` here as well for it to work.

```suggestion
const dataTypes = [ 'action', 'filter', 'slotFill' ];
```